### ISSUE TITLE
piping from helm template fails with empty objects

### ISSUE THREAD

**Issue by clux** [garethr/kubeval#64]
Suppose you have a chart feature that's conditionally included, like say cronjobs:

```
{{- range $v := $.Values.cronJobs }}
---
apiVersion: batch/v1beta1
kind: CronJob
...
{{- end }}
```

If you have a service not using this feature (not supplying any `Values.cronJobs`), then helm template will output (among other things):

```
---
# Source: base/templates/cronjobs.yaml

```

as the result for that template file, and `kubeval` will complain about `* Missing a kind key` for this part of the resource.

Is there a sensible way to ignore these failures? Is this maybe a helm template bug?

**Comment by garethr** [comment]
`kubeval` will ignore blank documents, but it's not currently ignoring documents without any non-comment content. This will need a quick fix in `kubeval`. Thanks for reporting. I'll take a run at this when I get a moment, but if anyone else wants to have a look at fixing beforehand just let me know.

**Comment by garethr** [comment]
This should be fixed in master via #79

**Comment by clux** [comment]
Thanks, works good here now :+1:
